spec/services/marketing_site_spec.rb

```diff
@@ -208,5 +208,14 @@
         it { expect(result).to eq(true) }
       end
     end
+
+    context 'with an anchor that makes the URL invalid' do
```

Comment:
Nit: Might suggest nesting this into the "with anchor" context above, so we don't need to redefine `result` ?

Reply:
whoops, that is my bad copy/pasting things around. willfix

Reply:
I didn't do exactly that, but modified the context "with a valid anchor" and moved this one underneath. Trying to avoid the pyramid of doom.